src/tool_help.c

```diff
@@ -405,3 +407,76 @@ void tool_list_engines(void)
   curl_slist_free_all(engines);
   curl_easy_cleanup(curl);
 }
+
+/* Functions for outputting cheat-sheet argument.
+   Cheat sheet data structure organized by heading and option. */
+static const struct cheat_table {
+  const char *head;
+  const char *opt;
+} cheat_items[] = {
+  {"Verbose", "-v, --trace-ascii file"},
+  {"Hide progress", "-s"},
```

Comment:
Maybe something like this?
A bit longer but I think its what people expect
```suggestion
  {"Hide progress, show errors", "-sS"},
```